Decred Wallet/Extensions/UIButton.swift

```diff
@@ -66,3 +66,12 @@ extension UIButton {
         }
     }
 }
+
+extension UIButton: XIBLocalizable {
```

Comment:
Can a similar extension be used for `UILabel` instead of creating a `UILabel` subclass?